pages/index.js

```diff
@@ -46,31 +44,19 @@ export default function Home({ accessToken }) {
         <meta name="description" content="Generated by create next app" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
       </Head>
+
       <main>
-        <div>
-          <MapBoxSidebar>
-            Longitude: {lng} | Latitude: {lat} | Zoom: {zoom}
-          </MapBoxSidebar>
-          <Map ref={mapContainer} />
-        </div>
+        <MapWrapper>
+          <Map
+            accessToken={accessToken}
+            lng={lng}
+            lat={lat}
+            zoom={zoom}
+            center={center}
+          />
+          <SearchPlaces handleSearchSubmit={handleSearchSubmit} />
```

Comment:
```suggestion
          <SearchPlaces onSearchSubmit={handleSearchSubmit} />
```
Then you should also adjust the prop in your `SearchPlaces` component.